src/io/common.jl

```diff
@@ -45,10 +45,18 @@ function check_network_data(data::Dict{String,Any})
     PMs.standardize_cost_terms(data)
 end
 
+function wrapto180(degrees::MultiConductorVector)
+    MultiConductorVector(wrapto180(degrees[:]))
+end
+
 function wrapto180(degrees)
     return degrees - 360*floor.((degrees .+ 180)/360)
 end
 
+function wraptopi(degrees::MultiConductorVector)
+    MultiConductorVector(wraptopi(degrees[:]))
+end
+
 function wraptopi(radians)
     return radians - 2*pi*floor.((radians .+ pi)/(2*pi))
 end
```

Comment:
If the original `wrapto180` and `wraptopi` aren't returning `MultiConductorVector`s, we should ensure that this is working in PowerModels.jl. This could be a problem somewhere else too.

Reply:
I cannot confirm that this is a problem in PowerModels, these functions both return MultiConductorVectors in their original incarnations. Unless you have an example where it fails I think we can remove the two new function declarations.

Reply:
This was a quick fix for another issue, that I am only now fully aware of when looking at it again. Take for example the following vector and operations
`a = MultiConductorVector(ones(Float64, 3))`
`floor.(a) .+ a`

This all works as expected. However, the solution dictionary contains vectors of the type
`va = MultiConductorVector(ones(Real, 3))`

Element-wise operations are not defined for the element type Real
`floor.(va) .+ va`

@ccoffrin is this the intended behaviour?

Reply:
Ok, I managed to track this bug down, it has to do with how broadcasting is implemented in Julia `> v"0.7.0-"`. I've made a PR for this fix in PowerModels, which will also hopefully fix the problem with `norm` in the Julia v0.6 travis tests.

Reply:
Great! The broadcasting magic went beyond my current understanding of Julia. I will remove the wrapper functions in common.jl in the meanwhile.

Reply:
Note that this will break most of the tests in transformer.jl

Reply:
That's okay, once PowerModels gets tagged with these broadcast fixes (@ccoffrin ), I will restart the travis runs, and make sure everything is really working